modules/formatTime.js

```diff
@@ -0,0 +1,13 @@
+var os = require('os');
+
+function formatTime(){
+    var uptime = os.uptime();
+    var seconds = (uptime % 60).toFixed(0);
+    var minutes = (uptime / 60 % 60).toFixed(0);
```

Comment:
are you sure that this will work correct? i tried for 1h 0m 56s (3600 + 0 + 56 = 3656) ;)